sunpy/timeseries/sources/norh.py

```diff
@@ -54,10 +54,42 @@ class NoRHTimeSeries(GenericTimeSeries):
     def __init__(self, data, header, units, **kwargs):
         super().__init__(data, header, units, **kwargs)
 
+    def plot(self, axes=None, **kwargs):
+        """
+        Plot the NoRH lightcurve TimeSeries from a pandas dataframe.
+
+        Parameters
+        ----------
+        axes : `matplotlib.axes.Axes`, optional
+            The axes on which to plot the TimeSeries. Defaults to current axes.
+        **kwargs : `dict`
+            Additional plot keyword arguments that are handed to `~matplotlib.axes.Axes.plot`
+            functions.
+
+        Returns
+        -------
+        `~matplotlib.axes.Axes`
+            The plot axes.
+        """
+        self._validate_data_for_plotting()
+        if axes is None:
+            axes = plt.gca()
+        plt.xticks(rotation=30)
+        data_lab = str(self.meta.get('OBS-FREQ').values()).replace('[', '').replace(
+            ']', '').replace('\'', '')
+        axes.plot(self.to_dataframe().index, self.to_dataframe(), label=data_lab, **kwargs)
+        axes.set_yscale("log")
+        axes.set_ylim(1e-4, 1)
+        axes.set_xlabel('Start time: ' + self.to_dataframe().index[0].strftime(TIME_FORMAT))
+        axes.set_ylabel('Correlation')
+        axes.legend()
+        return axes
+
     @peek_show
     def peek(self, **kwargs):
```

Comment:
This one as well. Did we not set this on enough peek methods?

Reply:
I think we decided that we wouldn't be adding it in if it had it before I think